python/lsst/pipe/tasks/multiBand.py

```diff
@@ -251,7 +292,7 @@ def __init__(self, schema=None, **kwargs):
                              fields set by this task.  If None, the source minimal schema will be used.
         @param[in] **kwargs: keyword arguments to be passed to lsst.pipe.base.task.Task.__init__
         """
-        CmdLineTask.__init__(self, **kwargs)
+        super().__init__(**kwargs)
```

Comment:
Before people get all super-excited, I think it should be pointed out that we have in the past formally *discouraged* the use of `super` except in "well-understood and designed class hierarchies" (or something like that).

Reply:
I was referencing a conversation I had with Nate where he pointed out that this is only sufficient because `CmdlineTask` doesn't have its own `__init__`.  When we mapped this out on Jim' whiteboard, we originally thought that the multiple inheritance for all the tasks would get complicated. All I'm saying is this miracle should be documented, and it should be obvious that we're calling `PipelineTask`'s init, which then calls `Task`'s init.

Reply:
We are explicitly using super here to deal with the multiple inheritance relations we have introduced. I am carefully inspecting all init method chains for tasks that will be in the super chain to ensure everything works. I will add a comment to the code.